iron-list.html

```diff
@@ -1297,6 +1297,10 @@
       this._itemsPerRow = this._itemWidth ? Math.floor(this._viewportWidth / this._itemWidth) : this._itemsPerRow;
     },
 
+    _isRTL: function() {
+      return window.getComputedStyle(this).direction === 'rtl';
```

Comment:
the styles are already computed in `updateViewportBoundaries` would you mind making _isRTL  a property and then updating the value in `updateViewportBoundaries`, e.g.:

```js
updateViewportBoundaries: function() {
  var styles = window.getComputedStyle(this);
  this._scrollerPaddingTop = this.scrollTarget === this ? 0 : parseInt(styles['padding-top'], 10);
  this._isRTL = Boolean(styles.direction === 'rtl');
...
}